react/features/base/ui/components/web/Input.tsx

```diff
@@ -177,7 +181,7 @@ const Input = React.forwardRef<any, IProps>(({
                 {icon && <Icon
                     { ...(iconClick ? { tabIndex: 0 } : {}) }
                     className = { cx(styles.icon, iconClick && styles.iconClickable) }
-                    onClick = { iconClick }
+                    { ...(iconClick ? { onClick: iconClick } : {}) }
```

Comment:
Why can't we let undefined be set here?

Reply:
I was under the impression a warning would show up in the console. Will revert it